pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -1035,3 +1030,24 @@ func getConfigDataFromHubURL(th *v1alpha1.TektonHub) (*Data, error) {
 
 	return data, nil
 }
+
+// checks target namespace, creates namespace if not found
+func (r *Reconciler) checkAndCreateNameSpace(ctx context.Context, th *v1alpha1.TektonHub) error {
+	if th.Spec.TargetNamespace != "openshift-pipelines" {
```

Comment:
@pratap0007 in kubernetes we use `tekton-pipelines` and in openshift `openshift-pipelines`

Reply:
Why do we need this check?

Reply:
To check the hub target namespace is same as tekton pipelines target namespace or not, if not then it creates the namespace